### PR TITLE
Update infra test for #5130

### PR DESCRIPTION
**Pull request by obabec**
### Type of change

- Enhancement / new feature

### Description

Updated infra test to check if address if working properly when it's connected to infra config which has omitted globalMaxSize. 
Address status is changed to ready even with NaN error on Agent so we have to test if address is working properly with sender&receiver attached.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [* ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by obabec** [comment]
@enmasse-ci run tests profile=systemtests testcase=isolated.infra.standard.InfraTestStandard test_only

**Comment by obabec** [comment]
@enmasse-ci run tests profile=acceptance

File: systemtests/src/test/java/io/enmasse/systemtest/isolated/infra/standard/InfraTestStandard.java [hunk 1]
```diff
@@ -162,6 +161,9 @@ void testCreateInfra() throws Exception {
         assertInfra(InfraConfiguration.broker(null, "512Mi", brokerTemplateSpec, "1Gi", "-Dsystemtest=property"),
                 InfraConfiguration.router(null, "256Mi", routerTemplateSpec, 1),
                 InfraConfiguration.admin(null, "512Mi", adminTemplateSpec));
+
+        clientUtils.assertCanConnect(exampleAddressSpace, new UserCredentials("test", "test"),
+                Collections.singletonList(exampleAddress), resourcesManager);
```
Comment (k-wall) [inline systemtests/src/test/java/io/enmasse/systemtest/isolated/infra/standard/InfraTestStandard.java:166]: this change does not actually validate the issue.

**Comment by obabec** [comment]
@enmasse-ci run tests profile=acceptance test_only

**Review by k-wall — APPROVED** [review]
Looks reasonable to me.

**Review by kornys — APPROVED** [review]

**Comment by obabec** [comment]
@enmasse-ci run tests profile=acceptance

**Comment by obabec** [comment]
@enmasse-ci run tests profile=acceptance

**Pull request merged**